.devcontainer.example/library-scripts/common-debian.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
It seems that we don't have any source file under MIT License in this repo.

Reply:
How should we maintain this file in the long run?

Reply:
My 2 cents is to keep it on another repo. We can just link to it.

Is this possibly coming already from a repo from Microsoft? That gets update as new versions come out?

Reply:
These Shell scripts yes, but rest of the files are tailored. Any preferences regarding external repo. Should it be my personal repository? Take into consideration that if it would be outside then it will be out of control.

Reply:
> How should we maintain this file in the long run?

If it would be here then I think manual process. README contains info on how it was more-or-less created.